src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -115,6 +115,12 @@
 	$pconfig['descr'] = $a_phase1[$p1index]['descr'];
 	$pconfig['nat_traversal'] = $a_phase1[$p1index]['nat_traversal'];
 	$pconfig['mobike'] = $a_phase1[$p1index]['mobike'];
+	$pconfig['gw_duplicates'] = $a_phase1[$p1index]['gw_duplicates'];
+
+	if (isset($a_phase1[$p1index]['gw_duplicates'])) {
+		$pconfig['gw_duplicates'] = true;
+	}
```

Comment:
I didn't get the idea here, first you attribute the value of `$a_phase1[$p1index]['gw_duplicates']` to `$pconfig['gw_duplicates']` and then, if former is set you attribute true to`$pconfig['gw_duplicates']`.  What are you trying to achieve?